python/ray/experimental/sgd/pytorch/utils.py

```diff
@@ -99,8 +194,15 @@ def validate(model, val_iterator, criterion, config):
             batch_time.update(time.time() - end)
             end = time.time()
 
-    stats = {"batch_time": batch_time.avg, "validation_loss": losses.avg}
-    stats.update(mean_accuracy=correct / total)
+            if config.get(TEST_MODE) and batch_idx == 0:
+                break
+
+    stats = {
+        BATCH_COUNT: batch_idx + 1,
+        "batch_time": batch_time.avg,
+        "validation_loss": losses.avg
+    }
+    stats.update(mean_accuracy=correct / total, mean_loss=losses.sum / total)
```

Comment:
why not just initialize these inline above?

Reply:
done, good idea